cmd/sops/main.go

```diff
@@ -211,6 +213,33 @@ func main() {
 				},
 			},
 		},
+		{
+			Name:  "updatekeys",
+			Usage: "update the keys of a SOPS file using the config file",
+			Flags: append([]cli.Flag{
+				cli.StringFlag{
+					Name:  "file, f",
+					Usage: `the file or directory that should be updated with the new keys. If providing a directory, SOPS will walk it and update all files in all subdirectories. '.git' directories and '.sops.yaml' files are ignored.`,
```

Comment:
Using a flag is different from the regular sops command line that just takes the file from the first non-flag argument. I think we should preserve it. 

As far as passing a whole directory, I wonder if that's needed, because ops can always use something along the line of `find . -name '*.yaml' -exec sops updatekeys {} \;`